### PR TITLE
adds anomatic tags

### PR DESCRIPTION
**Pull request by kesslerdev**

### CODE REVIEW

File: php/Makefile [hunk 1]
```diff
@@ -1,4 +1,6 @@
 TAG=$(PHP_VERSION)-$(PHP_VARIANT)
+CUST_TAG=$(shell echo $(TAG) | sed -e 's/^\([0-9]*\.[0-9]*\)\.[0-9]*/\1/g')
```
Comment (shouze) [inline php/Makefile]: ok for:
```shell
$ echo 7.3.21 | sed -e 's/^\([0-9]*\.[0-9]*\)\.[0-9]*/\1/g'
7.3
```
for
```shell
echo 7.3 | sed -e 's/^\([0-9]*\.[0-9]*\)\.[0-9]*/\1/g'
7.3
```

but for:
```shell
$ echo 7.3-rc | sed -e 's/^\([0-9]*\.[0-9]*\)\.[0-9]*/\1/g'
7.3-rc
```

is it intentional?

Comment (shouze) [reply php/Makefile]: In fact `sed -E 's/^([0-9]+\.[0-9]+).*/\1/'` is maybe more appropriate?

Comment (shouze) [reply php/Makefile]: -E is not POSIX compliant option 😭 so...
```shell
sed -e 's/^\([0-9]*\.[0-9]*\).*/\1/'
```
would to the trick

Comment (kesslerdev) [reply php/Makefile]: yes if no match keep the actual tag

**Pull request merged**